library_generation/utilities.sh

```diff
@@ -81,27 +81,31 @@ download_gapic_generator_pom_parent() {
 get_grpc_version() {
   local gapic_generator_version=$1
   local grpc_version
+  cd "${output_folder}"
   # get grpc version from gapic-generator-java-pom-parent/pom.xml
   download_gapic_generator_pom_parent "${gapic_generator_version}"
   grpc_version=$(grep grpc.version "gapic-generator-java-pom-parent-${gapic_generator_version}.pom" | sed 's/<grpc\.version>\(.*\)<\/grpc\.version>/\1/' | sed 's/^[[:space:]]*//;s/[[:space:]]*$//')
-  echo "$grpc_version"
+  cd ..
```

Comment:
If I use `popd` here, the value of directory stack somehow gets into `grpc_version`.
The grpc_version becomes `sdk-platform-java sdkplatform-java/output 1.58.0`.

Reply:
I think it's because the `stdout` of the function includes the output of `pushd` and `popd`.  I tried  `pushd $folder > /dev/null` and the units worked (and I could replicate the incorrect grpc version you mentioned if I don't use `> /dev/null`)

Reply:
cool, I'll try that.

Reply:
It works, thanks.